### ISSUE TITLE
Not Working in IPAD

### ISSUE THREAD

**Issue by Biplovkumar** [githuboftigran/rn-range-slider#24]
1. Version number :-  "rn-range-slider": "^1.2.8"
2. Crash on running in physical device.

Error :- 

![Screenshot 2019-09-03 at 5 37 35 PM](https://user-images.githubusercontent.com/30401136/64171975-90f34880-ce71-11e9-9cc6-d96d77acfe21.png)

**Comment by githuboftigran** [comment]
@Biplovkumar I can't see the error, could you please add more details? Like the error and steps to reproduce? BTW everything works normally on iPhones?

**Comment by githuboftigran** [comment]
@Biplovkumar, is this issue still relevant? Could you please download the latest version? If you can reproduce this, please send the crash log.

**Comment by rafalzawadzki** [comment]
Same issue, version 1.28. Happens on Iphone 5, 5c, iOS 10.

Stack:

```
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x000000000000000c

Crashed: com.apple.main-thread
0  <app>                      0x53e7e4 -[RangeSliderManager rangeSliderValueWasChanged:fromUser:] + 58 (RangeSliderManager.m:58)
1  <app>                      0x53e7e5 -[RangeSliderManager rangeSliderValueWasChanged:fromUser:] + 58 (RangeSliderManager.m:58)
2  <app>                      0x53d2eb -[RangeSlider checkAndFireValueChangeEvent:oldHigh:fromUser:] + 334 (RangeSlider.m:334)
3  <app>                      0x53cbab -[RangeSlider fitToMinMax] + 224 (RangeSlider.m:224)
4  <app>                      0x53ca93 -[RangeSlider setMin:] + 207 (RangeSlider.m:207)
5  <app>                      0x362ef7 __49-[RCTComponentData createPropBlock:isShadowView:]_block_invoke_6 + 287 (RCTComponentData.m:287)
6  <app>                      0x363763 __49-[RCTComponentData createPropBlock:isShadowView:]_block_invoke_2.86 + 315 (RCTComponentData.m:315)
7  <app>                      0x363935 __37-[RCTComponentData setProps:forView:]_block_invoke + 351 (RCTComponentData.m:351)
8  CoreFoundation                 0x1d388ebb __65-[__NSDictionaryM enumerateKeysAndObjectsWithOptions:usingBlock:]_block_invoke + 100
9  CoreFoundation                 0x1d388dc7 -[__NSDictionaryM enumerateKeysAndObjectsWithOptions:usingBlock:] + 156
10 <app>                          0x3638cb -[RCTComponentData setProps:forView:] + 353 (RCTComponentData.m:353)
11 <app>                          0x35d659 __44-[RCTUIManager flushUIBlocksWithCompletion:]_block_invoke + 1108 (RCTUIManager.m:1108)
12 <app>                          0x35d765 __44-[RCTUIManager flushUIBlocksWithCompletion:]_block_invoke.389 + 1133 (RCTUIManager.m:1133)
13 libdispatch.dylib              0x1cb41797 _dispatch_call_block_and_release + 10
14 libdispatch.dylib              0x1cb41783 _dispatch_client_callout + 22
15 libdispatch.dylib              0x1cb45d05 _dispatch_main_queue_callback_4CF + 902
16 CoreFoundation                 0x1d42fd69 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 8
17 CoreFoundation                 0x1d42de19 __CFRunLoopRun + 848
18 CoreFoundation                 0x1d3811af CFRunLoopRunSpecific + 470
19 CoreFoundation                 0x1d380fd1 CFRunLoopRunInMode + 104
20 GraphicsServices               0x1eb2bb41 GSEventRunModal + 80
21 UIKit                          0x22709a53 UIApplicationMain + 150
22 <app>                          0x89e5d main + 14 (main.m:14)
23 libdyld.dylib                  0x1cb6e4eb start + 2
```

**Comment by rafalzawadzki** [comment]
@githuboftigran Any idea if there is a solution to this?

**Comment by githuboftigran** [comment]
@rafalzawadzki , Will have a look soon!

**Comment by rafalzawadzki** [comment]
@githuboftigran I appreciate your time on this - we are having a substantial amount of crashes and consider changing the slider library for this reason.

Btw, I'd suggest correcting the title of this issue to smth like "Native crash on iOS 10" ;)

**Comment by Biplovkumar** [comment]
Any Update regarding this  ?

**Comment by githuboftigran** [comment]
@Biplovkumar , @rafalzawadzki just fixed in 1.3.1
